tasks/fh-build.js

```diff
@@ -13,28 +13,26 @@
  See the License for the specific language governing permissions and
  limitations under the License.
  */
-var path = require('path');
-var findup = require('findup-sync');
-var _ = require('underscore');
-var fs = require('fs');
-var _ld = require('lodash');
+var path = require('path')
+var findup = require('findup-sync')
+var _ = require('lodash')
+var fs = require('fs')
 
+module.exports = function (grunt) {
+  'use strict'
 
-module.exports = function(grunt) {
-  'use strict';
+  require('time-grunt')(grunt)
 
-  require('time-grunt')(grunt);
-
-  var tasksDir;
-  var dependencies = require('../package.json').dependencies;
-  var basepath = path.resolve(__dirname, '..');
+  var tasksDir
+  var dependencies = require('../package.json').dependencies
+  var basepath = path.resolve(__dirname, '..')
 
   for (var dependency in dependencies) {
     if (dependency.indexOf('grunt-') === 0) {
       tasksDir = findup(path.join('node_modules', dependency, 'tasks'),
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)